sonar-csharp-plugin/pom.xml

```diff
@@ -40,30 +40,34 @@
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <sonarAnalyzer.workDirectory>${project.build.directory}/analyzer</sonarAnalyzer.workDirectory>
     <documentationDirectory>${project.basedir}/src/main/resources</documentationDirectory>
+    <sonar.analyzer.commons>1.14.1.690</sonar.analyzer.commons>
```

Comment:
Does this have to be on 3 different places? Can we define the property in the root pom once?